File: C++Tutorial/src/myclass.cpp
#include "myclass.hpp"


MyClass::MyClass(int id, std::string nm){
  ID = id;
  name = nm;
}

std::string MyClass::change_name(std::string new_name){

  std::string old_name = name;
  name = new_name;

  return old_name + "->" + new_name;
}

File: PythonTutorial/TutorialLib/MyClass.py
class MyClass:
    name = ""
    ID = -1

    def __init__(self, ID, name):
        self.ID = ID
        self.name = name

    def get_id(self):
        return self.ID

    def get_name(self):
        return self.name

    def change_name(self, new_name):
        old_name = self.name
        self.name = new_name
        return old_name + " -> " + new_name
